pmd-kotlin/src/test/resources/net/sourceforge/pmd/lang/kotlin/cpd/testdata/comment.kt

```diff
@@ -1,3 +1,7 @@
+/*
```

Comment:
Isn't it weird to require those comments in test data files? We don't do that for other languages than Java, and I'm not sure what the point is. I feel like it could get in the way. Also in some other language modules, some test data files already have another license if they were copied from another project.

Reply:
No, I don't think it's weird. It's part of the project, whether it is used for tests or not, doesn't matter. Therefore the file should have the same license as the project...
We maybe should add license headers to the other files as well, but that's not a big priority.
That these files now pop up for kotlin is a side effect of using checkstyle for linting kotlin. We could consider integrating ktlint into the build, but I didn't go that far yet...

Reply:
Ok then, I'll merge tonight